packages/tools/src/utilities/math/polyline/calculateAreaOfPoints.ts

```diff
@@ -0,0 +1,19 @@
+import type { Types } from '@cornerstonejs/core';
+
+/**
+ * cCalculates the area of an array of `Point2` points using the shoelace algorithm.
+ */
+export default function calculateAreaOfPoints(points: Types.Point2[]): number {
```

Comment:
what is the unit for this return? we should document its unit

Reply:
In whatever units the points are in. Right now this is used for a calculation on canvas (on the fuse contour logic), but I can see it being used later for area calculation in other units. I'll doc this.